package.json

```diff
@@ -384,7 +384,7 @@
                 },
                 {
                     "command": "azureFunctions.createFunctionAppAdvanced",
-                    "when": "view == azureResourceGroups && viewItem =~ /microsoft.web/functionapp.*azureResourceTypeGroup/i",
+                    "when": "view == azureResourceGroups && viewItem =~ /microsoft.web/functionapp/i && viewItem =~ /azureResourceTypeGroup/i",
```

Comment:
Making it match https://github.com/microsoft/vscode-azurefunctions/blob/main/package.json#L382